tests/test_zarr_ipfs_partial.py

```
@@ -0,0 +1,461 @@
+        # Check full read result
+        assert results[3].to_bytes() == full_chunk_data, (
+            "Full read via get_partial_values failed"
+        )
+        print(f"Full Read: OK (Got {len(results[3].to_bytes())} bytes)")
+
+        # --- End: New Partial Values Tests ---
+
+        # Tests for code coverage's sake
+        assert await zhs_read.exists("zarr.json")
+        # __eq__
+        assert zhs_read == zhs_read
```

Pylint (3.3.7)

[refactor] Redundant comparison - zhs_read == zhs_read

(R0124)